docs/overview/Glossary.md

```diff
@@ -163,13 +161,15 @@ The device that is doing the encryption
 
 ### end-to-end-verifiable election
 
-An election is [end-to-end verifiable][election-guard-verifiability][^f1] if:
+An election is [end-to-end verifiable][election-guard-verifiability][^f1] if both are true:
 
-- **_Cast as intended_** -- _voters_ can verify that their own selections have been correctly recorded.
+- **_Cast as intended_** -- _voters_ can verify that their own selections have been correctly recorded; and
 - **_Counted as cast_** -- _anyone_ can verify that the recorded votes have been correctly tallied.
 
 ### extended base hash $\overline{Q}$
 
+A cryptographically unique piece of data generated from the [manifest](#manifest) and encrypted with the [joint public key](#joint-key) that is used to uniquely associate an encrypted ballot with a specific election.
```

Comment:
This desperately needed a description which I attempted to provide from data I found elsewhere, please verify.